### PR TITLE
Update Login/Signup pages

### PR DESCRIPTION
**Pull request by keca2**
Jan15 - NEW FORGOT PASSWORD PAGE:

image

UPDATED LOGIN:

image

OLD LOGIN:
image

UPDATED SIGNUP:
image

OLD SIGNUP:
image

### CODE REVIEW

**Comment by bailee-crandley** [comment]
Great news, I just ran it locally and it looks like the background handles screen resizing really well!

File: src/pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.login {
+    --background: url(../../assets/Walktober-graphic.png) center top;
+}
+
+.orDivider{
```
Comment (bailee-crandley) [inline src/pages/login/login.css:5]: The standard for css classes is kebab case. Could we change these to .or-divider?

**Review by bailee-crandley — APPROVED** [review]

**Comment by juroc95** [comment]
Looks amazing!

File: src/pages/login/login.css (hunk 1 above)
Comment (keca2) [reply src/pages/login/login.css]: Changed!

**Review by NathanMoes — COMMENTED** [review]
Seems good to me. Love the theme styling!

**Pull request merged**